src/EnergyPlus/SolarShading.cc

```diff
@@ -10440,13 +10439,14 @@ void SkyDifSolarShading(EnergyPlusData &state)
         state.dataSurface->Surface(SurfNum).ViewFactorGroundIR = 1.0 - state.dataSurface->Surface(SurfNum).ViewFactorSkyIR;
 
         if (state.dataSurface->SurfHasSurroundingSurfProperties(SurfNum)) {
+            int SrdSurfsNum;
+            Real64 SrdSurfsViewFactor = 0.0;
             SrdSurfsNum = state.dataSurface->SurfSurroundingSurfacesNum(SurfNum);
-            if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
-                state.dataSurface->Surface(SurfNum).ViewFactorSkyIR *= state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor;
-            }
-            if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor != -1) {
-                state.dataSurface->Surface(SurfNum).ViewFactorGroundIR *= state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor;
```

Comment:
The deleted section does not guaranty view factors identity because it proportionally adjusts the sky and ground surfaces view factors only.